.rdoc_options

```diff
@@ -1,8 +1,4 @@
 exclude:
```

Comment:
Given that it now has no values, maybe the key can be removed too?

Reply:
Updated!